.github/workflows/build.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: Build
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build static website
+        run: docker run --rm -v $PWD:/srv/jekyll -eJEKYLL_UID=$UID jekyll/builder:pages jekyll build --config _config.yml,_prod.yml
+      - name: Run htmlproofer
+        run: docker run --rm -v $PWD/_site:/site/about jekyll/builder:pages /usr/gem/bin/htmlproofer /site --url-ignore "/jupyter,/webclient/,/cell/,/tissue/,/,"
```

Comment:
This replaces https://github.com/IDR/idr.openmicroscopy.org/blob/ef5ab7ff7ce693ccb2c73dacfd233a48b0fe19ee/build.sh

Advantage of keeping the script is it's easy to run locally. Disadvantage is the commands are duplicated in two places.

Reply:
Thanks and agreeed. I'll work on a second version that consumes the script directly although I'll need to drop some of the arguments (`-it`) which are simply not compatible with the GH environment.